internal/oc/release.go

```diff
@@ -69,13 +44,13 @@ func (r *release) GetMCOImage(log logrus.FieldLogger, releaseImage string, relea
 	}
 	if releaseImageMirror != "" {
 		//TODO: Get mirror registry certificate from install-config
-		mcoImage, err = getMCOImageFromRelease(log, releaseImageMirror, pullSecret, true)
+		mcoImage, err = r.getMCOImageFromRelease(log, releaseImageMirror, pullSecret, true)
```

Comment:
What is the difference between releaseImageMirror and releaseImage?
Seems that both are fully qualified domain name of a release image no?
Also, seems that this code does the same for both.